sdk/core/abort-controller/README.md

```diff
@@ -99,6 +99,8 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
 decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot.
 You will only need to do this once across all repos using our CLA.
 
+If you'd like to contribute to this library, please read the the [contributing guide](../../../CONTRIBUTING.md) to learn more about how to build and test the code.
```

Comment:
Ahhh crap, I realized I added a duplicate "the" in my original suggestion, looks like it got merged in the Identity PR :/

```suggestion
If you'd like to contribute to this library, please read the [contributing guide](../../../CONTRIBUTING.md) to learn more about how to build and test the code.
```